coupled_laplace_problem/README.md

```diff
@@ -0,0 +1,44 @@
+Black-box coupled Laplace equation
+------------------------------------------
+## Overview
+
+What is preCICE? What is black-box coupling? What is this tutorial about?
+
+## Requirements
+* Version `9.3` or greater of `deal.II` TODO: Check for lower version
+
+* [preCICE](https://github.com/precice/precice/wiki#1-get-precice)
+
+## Compiling and running
+Similar to the example programs, run
+```
+cmake -DDEAL_II_DIR=/path/to/deal.II .
+```
+in this directory to configure the problem.
+You can switch between debug and release mode by calling either
+```
+make debug
+```
+or
+```
+make release
+```
+This command will generate two executables: one for the `coupled_laplace_problem` and one for the `dummy` participant.
+```
+make run
+```
+excutes the `coupled_laplace_problem`. In order to run the coupled simulation, execute
```

Comment:
You mean call both by `make run`? This might be difficult since I use the (deal.II default) predefined macro for the make target. Apart from that, I would propose to really require a manual execution in two terminals. Otherwise, the 'two different executable' aspect might be shadowed.

Reply:
Yes, simply don't allow `make run` at all, but tell users to start both codes explicitly. Can we avoid `make run` with the deal.II default?

Reply:
TODO: Delete `make run` target from README